doc/release-notes/bitcoin/release-notes-0.9.0.md

```diff
@@ -175,15 +175,15 @@ RPC:
 - Add 'dumpwallet' and 'importwallet' RPCs
 - 'keypoolrefill' gains optional size parameter
 - Add 'getbestblockhash', to return tip of best chain
-- Add 'chainwork' (the total work done by all blocks since the genesis block)
+- Add 'chainwoof' (the total woof done by all blocks since the genesis block)
   to 'getblock' output
 - Make RPC password resistant to timing attacks
 - Clarify help messages and add examples
 - Add 'getrawchangeaddress' call for raw transaction change destinations
 - Reject insanely high fees by default in 'sendrawtransaction'
 - Add RPC call 'decodescript' to decode a hex-encoded transaction script
 - Make 'validateaddress' provide redeemScript
-- Add 'getnetworkhashps' to get the calculated network hashrate
+- Add 'getnetwoofhashps' to get the calculated network hashrate
```

Comment:
Our Dear Leader hasn't yet deemed networks evil. Let's leave this line alone unless we receive further instructions from Him.

Reply:
fixed.

Reply:
Much fast. Such wow.